libs/c-ares/patches/001-fix_libcares_pc_path.patch

```diff
@@ -0,0 +1,16 @@
+--- a/libcares.pc.cmake
```

Comment:
I see this patch is added. Did you try to reach upstream if we can get rid of in the future? Maybe, it would be nice to have some slight description about this.

Reply:
Difference of $(STAGING_DIR)/usr/lib/pkgconfig/libcares.pc

1.18.1
```
prefix=/usr
exec_prefix=${prefix}/bin
libdir=${prefix}/lib
includedir=${prefix}/include
```

1.19.1 w/o patch
```
exec_prefix=/usr/bin
libdir=/usr/lib
includedir=/usr/include
```
1.19.1 with patch
```
prefix=/usr
#exec_prefix=${prefix}/bin
libdir=${prefix}/lib
includedir=${prefix}/include
```

Reply:
Additional information: ${prefix} is given externally.

Reply:
Ah, yes, if you give it to them in the c-ares configure, they might be fine without the patch. I'll give it a try.

Reply:
I still think this is reasonable.

Reply:
There's a similar PR in upstream https://github.com/c-ares/c-ares/pull/530

Reply:
CMAKE build, so it depends on libcares.pc.cmake.

Reply:
Past discussions about pkg-config.

https://github.com/openwrt/openwrt/pull/1615

Reply:
well, main question is if it can be fixed on upstream side so we won't have to keep this patch forever...

Reply:
I know that some openwrt packages have a sed replacement method.

Reply:
It seems complicated to convince pkg-config to use "${prefix}" instead of absolute paths. In normal use, absolute paths do not cause problems.

Reply:
Remove patch. Changed to replace with sed.

Reply:
Looks much better this time, but my original proposal still remains. In the commit message, there is no sight why it is necessary, because from the commit, it looks like usual package bump, which actually is not as there is added InstallDev section. One or two sentence would be handy as it helps in the future to look, why it was added (``git blame``), and it helps to understand it.

Reply:
OK. Added to the commit message.

Reply:
Awesome! 👏